src/tzstats/tzstats_reward_provider_helper.py

```diff
@@ -64,17 +65,26 @@ def get_rewards_for_cycle(self, cycle, expected_rewards=False):
             raise ApiProviderException('GET {} {}'.format(uri, resp.status_code))
 
         resp = resp.json()[0]
-        if expected_rewards:
+        if rewards_type.isEstimated():
             root["total_reward_amount"] = int(1e6 * float(resp[idx_income_expected_income]))
```

Comment:
@dansan566 we have to make a reminder to talk to Alexander Eichhorn about the wording here. After our discussion I am inclined to say that the expectation by the client is a smooth working bakery and the "expected" API call from tzstats is in fact an "estimated"